libpod/info.go

```diff
@@ -128,6 +128,12 @@ func (r *Runtime) hostInfo() (map[string]interface{}, error) {
 	}
 	info["hostname"] = host
 	info["eventlogger"] = r.eventer.String()
+
+	uidmappings, err := rootless.ReadMappingsProc("/proc/self/uid_map")
+	if err != nil {
+		return nil, errors.Wrapf(err, "error reading uid mappings")
+	}
+	info["uidmap"] = uidmappings
```

Comment:
I think we should also include "/proc/self/gid_map"

Reply:
I agree